rllib/agents/qmix/qmix_policy.py

```diff
@@ -143,6 +143,7 @@ def forward(self,
         return loss, mask, masked_td_error, chosen_action_qvals, targets
 
 
+# TODO: Make this a TorchPolicy child.
```

Comment:
Can you use `TODO(sven)` or something like that? The point of this is so that people looking at the TODO in the future know who wrote it and can talk to them.

Reply:
Sure.

Reply:
Done.